### DISCUSSION TITLE
Cooperation gamemode

### DISCUSSION THREAD

**Discussion by quarties** [Try/OpenGothic#391] [Ideas]
I just discovered this project and I'm impresses of what you did here 😲 Would it be possible to create cooperation gamemode in which more than one players are able to progress the game? I heard that G1 had implemented the network layer, but they ditched it. Also there are proejcts like Gothic 2 Online, but it's a closed project.

**Comment by Sashimiyo** [comment]
I would like to reactivate this thread and consider a "simple" way to add and render a friends hero. No shared instances and fancy stuff but only a small server sharing each heros position data to each clients.
